libbeat/management/management.go

```diff
@@ -152,30 +152,26 @@ func defaultModeConfig() *modeConfig {
 	}
 }
 
-// nilManager, fallback when no manager is present
-type nilManager struct {
+// fallbackManager, fallback when no manager is present
```

Comment:
Looks like this file has a bunch of unused types and variables we can remove (see above).

Reply:
I'll clean them up!